test/files/jvm/scala-concurrent-tck.scala

```diff
@@ -90,6 +90,25 @@ trait FutureCallbacks extends TestBase {
     promise.success(-1)
   }
 
+  def stressTestNumberofCallbacks(): Unit = once {
```

Comment:
@ichoran What's your recommendation for doing performance regression testing on the stdlib? I'd like to add some JMH benches here but I see none around.